src/CrudPanelManager.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Backpack\CRUD;
+
+use Backpack\CRUD\app\Http\Controllers\Contracts\CrudControllerContract;
+use Backpack\CRUD\app\Http\Controllers\CrudController;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanel;
+use Illuminate\Support\Facades\Facade;
+
+final class CrudPanelManager
+{
+    private array $cruds = [];
+
+    private ?string $currentlyActiveCrudController = null;
+
+    private $requestController = null;
+
+    public function crud(CrudControllerContract $controller): CrudPanel
```

Comment:
```suggestion
    private function getOrCreateCrudPanelInstance(CrudControllerContract $controller): CrudPanel
```

Let's make it private and rename it to `getOrCreateCrudPanelInstance()`